packages/cli/src/util/client.ts

```diff
@@ -1,21 +1,19 @@
 import { MedplumClient, MedplumClientOptions } from '@medplum/core';
 import { FileSystemStorage } from '../storage';
 
-export async function createMedplumClient(options: MedplumClientOptions): Promise<MedplumClient> {
-  const baseUrl = options.baseUrl ?? process.env['MEDPLUM_BASE_URL'] ?? 'https://api.medplum.com/';
-  const fhirUrlPath = options.fhirUrlPath ?? process.env['MEDPLUM_FHIR_URL_PATH'] ?? '';
-  const accessToken = options.accessToken ?? process.env['MEDPLUM_CLIENT_ACCESS_TOKEN'] ?? '';
-  const tokenUrl = options.tokenUrl ?? process.env['MEDPLUM_TOKEN_URL'] ?? '';
-  const authorizeUrl = options.authorizeUrl ?? process.env['MEDPLUM_AUTHORIZE_URL'] ?? '';
-  const fetchApi = options.fetch ?? fetch;
+export async function createMedplumClient(options: MedplumClientOptions, profileName?: string): Promise<MedplumClient> {
+  const profile = profileName ?? 'default';
```

Comment:
@jamestouri and I discussed this a little bit.  There isn't a great obvious term.  I agree that `profile` is overloaded, but also feel that this is a sufficiently different operating context that it might be ok.

Alternatives we discussed:
* `-c` / `--config` - just make it explicit that we're using a different config file
* `-c` / `--context` - based on CDK uses `-c` / `--config`
* `-i` / `--identity` - based on SSH `-i identity_file`
* `--persona` - although this implies "person", and the more common case will be machine-to-machine